lib/Plerd/Post.pm

```diff
@@ -130,9 +130,13 @@ sub _build_published_filename {
 sub _build_uri {
     my $self = shift;
 
+    my $base_uri = $self->plerd->base_uri;
+    if ($base_uri =~ /,[^\/]$/) {
```

Comment:
If you could be so kind as to zap the stray comma character in this line, this looks shippable!